nextflow.config

```diff
@@ -72,7 +72,7 @@ includeConfig 'conf/base.config'
 
 // Container slug. Stable releases should specify release tag!
 // Developmental code should specify :dev
-process.container = 'nfcore/ampliseq:1.1.2'
+process.container = 'nfcore/ampliseq:1.1.3'
 
 // Load base.config by default for all pipelines
 includeConfig 'conf/base.config'
```

Comment:
```suggestion
```

Reply:
True. Unfortunately I cannot commit your suggestion, could you do it? Thanks.